lib/platform/github/index.ts

```diff
@@ -1671,14 +1671,28 @@ export async function getVulnerabilityAlerts(): Promise<VulnerabilityAlert[]> {
       }
     }
   }`;
-  let alerts: VulnerabilityAlert[] = [];
+  let vulnerabilityAlerts: {
+    node: VulnerabilityAlert;
```

Comment:
Was this a bug?

Reply:
No, it's just that the try/catch was broken into two so there was a need to have two `let` types.